tests/standalone/e2e/ilab_standalone_rhoai_test.go

```diff
@@ -450,6 +527,20 @@ func GetTestServiceAccount() (string, bool) {
 	return data_key, exists
 }
 
+// GetTimeout expects a timeout specified in minutes
+func GetTimeOut() (string, bool) {
```

Comment:
I don’t believe a dedicated function is necessary for this. Since the caller needs to check for exists anyway, you could simply use return os.LookupEnv("TEST_RUN_TIMEOUT_MINUTES") directly, or remove it entirely and handle it in the caller.

Reply:
Yeah agree, this was more to be consistent with the rest of the codebase on how env vars are being fetched, but I don't think a dedicated function is necessary for those instance either 🤷🏾 

Don't have a strong opinion, amended.